### ISSUE TITLE
[RFM] - pycharm professional

### ISSUE THREAD

**Issue by deva1003** [chocolatey/chocolatey-package-requests#176]
looking for latest version of pycharm professional to be packaged as chocolatey. Thanks!

**Comment by AdmiringWorm** [comment]
I'm guessing this is the package you are looking for?
https://chocolatey.org/packages/Pycharm

Please try reaching out to the maintainer of that package first (look for the `Contact Maintainers` link), before we can open this one up for new maintainers of that package.

Thanks

**Comment by AdmiringWorm** [comment]
nvm, I see this is a duplicate of issue #115
